Makefile

```diff
@@ -240,41 +242,28 @@ clean_certs:
 ###                  Formatting, linting, and vetting                       ###
 ###############################################################################
 
-# https://github.com/cometbft/cometbft/pull/1925#issuecomment-1875127862
-# Revisit using format after CometBFT v1 release and/or after 2024-06-01.
-#format:
-#	find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
-#	find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/cometbft/cometbft
-#.PHONY: format
-
-#? lint: Run latest golangci-lint linter
+#? lint: Lint, format and fix typos
 lint:
-	@echo "--> Running linter"
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
+	@echo "--> Linting"
+	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run --fix
```

Comment:
Are you sure you want to automatically fix here? I think I liked it fails when there are problems.

Reply:
> Are you sure you want to automatically fix here?

Why would you not prefer the linter to fix the problem for you? `golangci-lint` will fail if the linter doesn't provide `AutoFix` https://golangci-lint.run/usage/linters/, which is largely still the case for many linters.

Reply:
I agree that fixing is the right thing to do. If it causes issues down the line, we'll revert to no-fixing.